### PR TITLE
Fix share link to image url

### PR DESCRIPTION
**Pull request by xamfy**

### CODE REVIEW

File: src/lightbox/LightboxContainer.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { connectActionSheet } from '@expo/react-native-action-sheet';
 import type { Actions, Auth, Message, ImageResource } from '../types';
 import connectWithActions from '../connectWithActions';
 import { getAuth } from '../selectors';
+import { getCurrentRealm } from '../account/accountSelectors';
```
Comment (jainkuniya) [inline src/lightbox/LightboxContainer.js]: you can import `getCurrentRealm` from `../selectors` too.
As it exports all selectors.
https://github.com/zulip/zulip-mobile/blob/master/src/selectors.js

File: src/lightbox/LightboxContainer.js [hunk 2]
```diff
@@ -62,7 +63,7 @@ class LightboxContainer extends PureComponent<Props, State> {
   handleOptionsPress = () => {
     const options = constructActionSheetButtons();
     const cancelButtonIndex = options.length - 1;
-    const { showActionSheetWithOptions, src, auth } = this.props;
+    const { showActionSheetWithOptions, src, realm, auth } = this.props;
```
Comment (borisyankov) [inline src/lightbox/LightboxContainer.js]: You can get the realm from the `auth` structure, no need to pass another parameter.

```
export type Auth = {
  realm: string,
  apiKey: string,
  email: string,
};
```

Comment (xamfy) [reply src/lightbox/LightboxContainer.js]: got it

**Comment by borisyankov** [comment]
Looks good!

**Pull request merged**